src/pushsource/_impl/model/vms.py

```diff
@@ -74,6 +74,16 @@ class VMIPushItem(PushItem):
     boot_mode = attr.ib(type=BootMode, default=None, validator=optional(in_(BootMode)))
     """Boot mode supported by the image (if known): uefi, legacy, or hybrid (uefi + legacy)."""
 
+    marketplace_title_template = attr.ib(type=str, default=None, validator=optional_str)
+    """The template is of the form used by ``str.format``, with available keywords being all of
+    the documented fields on ``VMIRelease`` and ``AMIRelease`` classes."""
+
+    @property
+    def marketplace_title(self) -> str:
+        if not self.marketplace_title_template:
```

Comment:
It seems that `self.release` is allowed to be `None`, so should that also cause it to go into this branch and return an empty string?

Reply:
You're right, good catch! I've patched it accordingly.